A side effect of altwin:meta_win is that Alt keys are modified.
altwin options are listed first so that other options, like
lv3:lwin_switch, can modify Alt keys.  Closes: #385355
Thanks Ben Mueller

Submitted upstream: https://bugs.freedesktop.org/show_bug.cgi?id=8124

Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part
+++ xkb-data/rules/base.o_s.part
@@ -1,3 +1,10 @@
+  altwin:menu		=	+altwin(menu)
+  altwin:meta_alt	=	+altwin(meta_alt)
+  altwin:meta_win	=	+altwin(meta_win)
+  altwin:left_meta_win	=	+altwin(left_meta_win)
+  altwin:super_win	=	+altwin(super_win)
+  altwin:hyper_win	=	+altwin(hyper_win)
+  altwin:alt_super_win	=	+altwin(alt_super_win)
   grp:switch		=	+group(switch)
   grp:lswitch		=	+group(lswitch)
   grp:win_switch	=	+group(win_switch)
@@ -38,13 +45,6 @@
   ctrl:ctrl_ac		=	+ctrl(ctrl_ac)
   ctrl:ctrl_aa		=	+ctrl(ctrl_aa)
   ctrl:ctrl_ra		=	+ctrl(ctrl_ra)
-  altwin:menu		=	+altwin(menu)
-  altwin:meta_alt	=	+altwin(meta_alt)
-  altwin:meta_win	=	+altwin(meta_win)
-  altwin:left_meta_win	=	+altwin(left_meta_win)
-  altwin:super_win	=	+altwin(super_win)
-  altwin:hyper_win	=	+altwin(hyper_win)
-  altwin:alt_super_win	=	+altwin(alt_super_win)
   compose:ralt		=	+compose(ralt)
   compose:lwin		=	+compose(lwin)
   compose:rwin		=	+compose(rwin)
